### ISSUE TITLE
No Void Rift Spawning

### ISSUE THREAD

**Issue by SlvrKnight96** [Azanor/thaumcraft-beta#1424]
I have run into a problem where no matter how much flux is in a chunk, I cannot get a void rift to spawn after a long period of time, even with hundreds or thousands of units of flux. It makes it impossible to get void seeds. I have only encountered this problem in my creative worlds so far where I am trying to test the void siphon, but I cannot get a rift to spawn in the first place.

**Comment by wamc2017** [comment]
Which version of Thaumcraft are you using? Is the area maybe protected through another mod? (FTB Utilities chunk claiming comes to my mind)

**Comment by SlvrKnight96** [comment]
I am using Beta 24. I am using Forge with a custom modpack with very few mods, including Tinker's Construct, Twilight Forest, and JEI. I have been able to spawn rifts in my main world using the same version, but it's been impossible to do the same in the creative worlds even after waiting an hour. I figured switching over to game mode 0 would help, but nothing works in my test worlds. I should try again and confirm that it even works in my main world still.

**Comment by wamc2017** [comment]
Hmm, works fine with only Thaumcraft installed on my side, do you have a modlist?

**Comment by Minamikaze392** [comment]
Just a guess. Are there cloud blocks high in the sky which is added by some mods?

**Comment by SlvrKnight96** [comment]
There aren't any cloud blocks in the sky that I am aware of. Also, if your suggestion is that the rift opened and I'm just not seeing it, I don't see the flux go down when I am monitoring it on the thaumometer. It doesn't look like one is opening on some of my worlds. Also, the modlist I am using includes:
Baubles
Mantle
Tinker's Construct
Thaumcraft
The Twilight Forest

**Comment by Azanor** [comment]
Do you maybe have wussMode on in your config?

**Comment by SlvrKnight96** [comment]
No, wussMode is not on in my config. I checked that when I first encountered the problem, but that is not the case. Is there any settings that differ between worlds that are created starting in creative mode as oppose to worlds that start in adventure?

**Comment by SlvrKnight96** [comment]
One suggestion I have, once the problem of flux rift spawning has been solved, would be the possibility of opening flux rifts on purpose with void seeds. I want to be able to create a farm that emits flux into the aura at about the rate that a rift consumes flux and then siphon the rift into void seeds, but there are no safe methods of spawning rifts without dumping hundreds and even thousands of units of flux into a chunk, which spreads to all of the surrounding chunks.

**Comment by Hubry** [comment]
Did you disable daylight cycle? I remember running into this problem already - daylight cycle seems required for flux to do anything at all.

**Comment by SlvrKnight96** [comment]
I did disable the daylight cycle. I'll try allowing it and see if that changes anything. Thanks for the suggestion!

**Comment by SlvrKnight96** [comment]
So I reenabled the daylight cycle, but it still doesn't summon rifts even after a long time. The flux now spreads to other chunks and vis will regen, unlike before. I'm not sure if Azanor changed something to make the rift event way less likely in one of the recent versions.

**Comment by Azanor** [comment]
Nope, I have not
